ts/features/wallet/components/WalletCardsContainer.tsx

```diff
@@ -64,17 +69,22 @@ const WalletCardsContainer = () => {
     }
     return (
       <View testID="walletCardsContainerTestID" style={IOStyles.flex}>
-        <ItwWalletCardsContainer />
+        {!isWalletInstanceStatusFailure && <ItwWalletCardsContainer />}
```

Comment:
We are conditionally rendering `ItwWalletCardsContainer` based on `isItwEnabledSelector`. We should move that outside the component, just like we are doing with `isWalletInstanceStatusFailure` here.
Or, we can move this inside the component, but in that case we are going to initialize some code before the early `null` return, which is not optimal.
What do you think?